test/unit/puppetlabs/services/ca/ca_testutils.clj

```diff
@@ -50,7 +50,11 @@
    :signeddir             (str cadir "/signed")
    :serial                (str cadir "/serial")
    :ruby-load-path        jruby-testutils/ruby-load-path
-   :gem-path              jruby-testutils/gem-path})
+   :gem-path              jruby-testutils/gem-path
+   :compile-masters       []
+   :compile-master-serials-path (str cadir "/cm-serials.txt")
+   :infra-crl-path (str cadir "/infra-crl.pem")
```

Comment:
spacing's off

Reply:
Did you mean that for infra-crl-path? Whats the convention when keyname is long like in case of compile-master-serials-path?